src/compas/scene/graphobject.py

```diff
@@ -60,13 +58,7 @@ def __init__(self, graph, **kwargs):
 
     @property
     def graph(self):
-        return self._graph
-
-    @graph.setter
-    def graph(self, graph):
```

Comment:
These setters I removed for now, because the `item` in `SceneObject` is currently read-only